src/ghostfat.c

```diff
@@ -33,9 +33,9 @@
 #include "board_api.h"
 #include "uf2.h"
 
-// use as: GF_DEBUG_PRINT(("foo %d", x));
+// use as: GF_DEBUG_PRINT("foo %d", x);
 #if defined(DEBUG)
-  #define GF_DEBUG_PRINT(x) do { printf x; fflush(stdout); } while (0)
+  #define GF_DEBUG_PRINT(x) do { printf(x); fflush(stdout); } while (0)
```

Comment:
Similarly, the lack of parenthesis here was also intentional.

Reply:
we could just use the `VA_ARGS` macros instead

Reply:
I removed the unused macro.  Later can define preferred debug output methods ... this was just a hack I needed for a short time.